alchemy-web/src/content/docs/providers/cloudflare/worker.mdx

```diff
@@ -5,80 +5,179 @@ description: Learn how to deploy, configure, and manage Cloudflare Workers using
+  },
+  observability: {
+    enabled: true                         // Enable worker logs / observability (default: true)
+  },
+  // Upload source maps to improve stack traces (set to false to disable)
+  sourceMap: true,
+  // Optional runtime limits — increase only when necessary after monitoring
+  limits: {
+    cpu_ms: 50_000                        // Max CPU time in ms (default: 30_000)
   },
+  // Other Advanced 🧪 & commonly used options
+  placement: { mode: "smart" },          // Smart placement to optimize network placement for latency
+  assets: { path: "./public", run_worker_first: false }, // Static assets configuration
+  crons: ["0 0 * * *"],                  // Scheduled cron triggers (standard cron syntax)
+  eventSources: [                        // Background event sources (queues, streams)
+    { queue: taskQueue, settings: { batchSize: 15, maxConcurrency: 3 } }
+  ],
+  routes: ["api.example.com/*"],          // Route patterns to bind the worker
+  domains: ["example.com"],               // Custom domains to bind (can be objects with zoneId/adopt)
+  version: "pr-123",                      // Publish as preview version with label
+  dev: { port: 8787 },                    // Local development options (port, remote, tunnel)
+  tunnel: true,                           // Expose the worker via a Cloudflare Tunnel.
+  namespace: await DispatchNamespace("my-dispatch-ns", {
+    namespace: "production-dispatch"
+  }),                    // Deploy to dispatch namespace (string | DispatchNamespace)
+  noBundle: false,                        // Disable bundling (deploy files as-is)
+  rules: [{ globs: ["**/*.wasm"] }],      // Additional bundle rules
 });
 ```
 
-Access in `./src/api.ts` by importing `cloudflare:workers`:
+:::tip
+Monitor your CPU usage with Cloudflare Analytics to understand if increased limits are necessary. Start with default limits and scale up only when needed to optimize costs.
+:::
 
+Access bindings using modern `env` import or traditional parameter approach:
 ```ts
 import { env } from "cloudflare:workers";
 
 export default {
-  async fetch() {
+  async fetch(request: Request, env: Env) {
     await env.CACHE.get("key");
+    return new Response("OK");
   },
 };
 ```
 
-Or via the `env` parameter:
-
+Global scope initialization (when appropriate):
 ```ts
+import { env } from "cloudflare:workers";
+import ApiClient from "example-api-client";
+
+const apiClient = new ApiClient({ apiKey: env.API_KEY });
+
 export default {
-  async fetch(request, env) {
-    await env.CACHE.get("key");
+  async fetch(request: Request) {
+    return new Response("Initialized with API key");
   },
 };
 ```
 
-### Infer Worker Env Types
-
-Binding types can be inferred from the `worker` in your `alchemy.run.ts` script:
-
+:::tip
+Use global scope initialization for expensive operations like database connections or API client setup. Avoid initializing per-request to minimize cold start times and improve performance.
+:::
```

Comment:
nit: it's module scope, not global scope

Gotta be careful here - IO is not allowed at the module scope.